pic-sure-resources/pic-sure-resource-api/src/main/java/edu/harvard/dbmi/avillach/domain/QueryRequest.java

```diff
@@ -38,6 +39,8 @@
 				"        \"fields\": []\n" +
 				"    }\n" +
 				"}")
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Gotta figure out how to sequence this deploy so that things don't explode. I _think_ we make all the institutions deploy, and then we update the common area. Jackson should ignore the missing common area UUID field until we start supplying it.

If we deployed the common area first, it would send QueryRequest objects with a field that the institutes don't know about, which would cause Jackson to explode.

I want to make sure we think about this and make sure we don't yeet a breaking change into a critical object.

Reply:
If we versioned our requests (in a header value or something), then we could add logic to check for backwards compatibility. But, might not be worth it at this point if we can otherwise avoid the issue.